developer_guides/topology2/topology2.rst

```diff
@@ -1332,6 +1332,53 @@ You can use the ``-P`` switch to convert a 2.0 configuration file to the 1.0 con
 
    alsatplg <-D args=values> -P input.conf -o output.conf
 
+Split topologies
+****************
+
+Linux kernel can load multiple topologies, a topology for a single function.
+This feature is useful to support SDCA setups with standardized components. And no need to create topologies
+for every new product. To achieve this, you need to split the topology into multiple tplg files.
+The split topology files should be named as follows:
+
+.. code-block:: bash
+
+        sof-<platform>-<function>-id<BE id number>.tplg
+
+Currently <platform> is only needed for the DMIC function.
```

Comment:
....and not needed for SDCA functions in general.

Reply:
updated